core/src/main/java/net/akami/yggdrasil/display/ActionBarSpellDisplayer.java

```diff
@@ -0,0 +1,49 @@
+package net.akami.yggdrasil.display;
+
+import org.spongepowered.api.Sponge;
+import org.spongepowered.api.entity.living.player.Player;
+import org.spongepowered.api.text.Text;
+import org.spongepowered.api.text.chat.ChatTypes;
+import org.spongepowered.api.text.format.TextColors;
+
+import java.util.*;
+
+public class ActionBarSpellDisplayer {
+    private Map<UUID, List<Text>> currentSequence = new HashMap<>();
```

Comment:
You can use directly a `Map<UUID, Text>` and use [Text#concat](https://jd.spongepowered.org/7.0.0/org/spongepowered/api/text/Text.html#concat-org.spongepowered.api.text.Text-) instead of a `Map<UUID, List<Text>>`